chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserApiType.kt

```diff
@@ -1,28 +1,13 @@
 package com.pusher.chatkit.users.api
 
-import com.google.gson.annotations.SerializedName
 import com.pusher.chatkit.CustomData
-import com.pusher.chatkit.presence.Presence
-import com.pusher.chatkit.util.dateFormat
-import java.util.Date
 
 internal data class UserApiType(
     val id: String,
-    private val createdAt: String,
-    private val updatedAt: String,
-
+    val createdAt: String,
+    val updatedAt: String,
     val name: String?,
-    @SerializedName("avatar_url")
-    val avatarURL: String?,
+    val avatarUrl: String?,
     val customData: CustomData?,
-    private var online: Boolean = false
-) {
-    var presence: Presence
-        get() = if (online) Presence.Online else Presence.Offline
-        set(value) {
-            online = value === Presence.Online
-        }
-
-    val created: Date by lazy { dateFormat.parse(createdAt) }
-    val updated: Date by lazy { dateFormat.parse(updatedAt) }
-}
+    var online: Boolean = false
```

Comment:
that var should be removed, it doesn't come from the backend API, moreover never a `var` should be used for our data classes

Reply:
hmm. Have removed. Does this come for the presence subscriptions? Is it assumed that when the current user is connected this then becomes "online" and we'll handle it at a different point?